src/components/common/ThemeToggle/index.tsx

```diff
@@ -24,6 +24,15 @@ const ThemeToggle = () => {
   const switchPos = themeToSwitch[theme];
   const currAltText = `Icon representing ${theme} theme is on.`;
 
+  useEffect(() => {
+    const metaThemeColor = document.querySelector('meta[name="theme-color"]');
```

Comment:
you should add a comment explaining that this is for styling the ios safari menu bar thingy

Reply:
Can we also change the default theme color to ACM's blue or something? that way it shows a blue line on the Discord link preview. it should still set it to the background color when the JavaScript runs